_shared/project/docker-compose.yml

```diff
@@ -1,12 +1,14 @@
 services:
   {% if cookiecutter.get("postgres") == "yes" %}
   postgres:
-    image: postgres:{{ cookiecutter.get("__postgres_version", "11.5-alpine") }}
+    image: postgres:{{ cookiecutter.get("__postgres_version", "15.3-alpine") }}
     ports:
       - "127.0.0.1:{{ cookiecutter['__postgres_port'] }}:5432"
     healthcheck:
       test: ["CMD", "pg_isready", "-U", "postgres"]
       interval: 1s
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: trust
```

Comment:
Postgres 15 refuses to start up without this envvar

Reply:
I was checking and we are disabling the password requirement with this.

I thought that maybe we'd need to update some of the database URLs but we already do not include a password there (and I reckon rely on the fact that the default was the same as the user, postgres/postgres?)